packages/config/src/discovery/ProjectDiscovery.ts

```diff
@@ -965,10 +992,11 @@ const roleDescriptions: { [key in StackRole]: string } = {
 }
 
 export function formatAsBulletPoints(description: string[]): string {
-  // Temporarily, for clean diffs, disable bullet points.
-  // It will be enabled as part of upcoming stories.
-  return description.join(' ')
-  // return description.length > 1
-  //   ? description.map((s) => `* ${s}\n`).join('')
-  //   : description.join(' ')
+  return description.length > 1
+    ? description.map((s) => `* ${s}\n`).join('')
+    : description.join(' ')
+}
+
+export function trimTrailingDots(s: string): string {
+  return s.replace(/\.*$/, '')
```

Comment:
I would feel more comfortable if this was a simple loop going backwards and creating a substring. LLM can easily rewrite it. Regex = Bad

Reply:
I think that code clarity beats more complex implementation here (looping, slicing, ...). This is one of the simplest regexes possible, you just know it works. But looping, slicing, requires much more analysis (are indexes used for slicing correct, are the zero based, what if it's an empty string, etc. etc.) I prefer to keep it like this.